index.html

```diff
@@ -1495,9 +1495,10 @@
 
 * The <dfn data-export="">right to access</dfn> [=data=] about oneself.
 
-This right includes being able to discover data about oneself (implying no databases are kept
-secret) and to review
-what information has been collected or inferred.
+This right includes both being able to review what information has been collected or inferred about
+oneself and being able to discover what [=actors=] have collected information about oneself. As a
+result, databases must not be kept secret and data collected about [=people=] must be meaningfully
```

Comment:
```suggestion
result, databases cannot be kept secret and data collected about [=people=] needs to be meaningfully
```

Just to avoid the normative language in a passive-voice context. I don't think this loses any meaning?